Sources/_NIO1APIShims/NIO1APIShims.swift

```diff
@@ -531,3 +531,6 @@ extension WebSocketFrameDecoder {
         self.init(maxFrameSize: maxFrameSize)
     }
 }
+
+@available(*, deprecated, renamed: "NIOWebSocketClientUpgrader")
+typealias NIOWebClientSocketUpgrader = NIOWebSocketClientUpgrader
```

Comment:
This is not a NIO 1 shim: we have to carry this name with us for the rest of the project. It must be imported and available alongside any import of NIOWebSocket. Can you move this into `NIOWebSocketClientUpgrader.swift`?

Reply:
Done.